my_microservice/core/greeting.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains functionalities for greeting persons."""
+
+from datetime import datetime
+import random
+from ..models import GreetingExpression, Greeting
+
+GREETINGS_EXPRESSIONS = [
+    GreetingExpression(expression="Kaliméra", language="Greek", isinformal=False),
+    GreetingExpression(expression="Yassou", language="Greek", isinformal=True),
```

Comment:
This hurts me, φίλε.

Reply:
Also consider Γεια σας (Geia sas/Yassas*) as another formal greeting 😉

Reply:
Haha, if you like please feel free to add it the correct spelling 😄